app/styles/partials/_info.scss

```diff
@@ -248,4 +248,58 @@ $packages: (
     width: 30px;
     order: 1;
   }
-}
\ No newline at end of file
+}
+
+.contributors img {
+  border-radius: 50%;
+  border: 2px solid #fff;
+  width: 30px;
+  margin-right: 5px;
+}
+
+/* Tooltip container */
+.tooltip {
+  position: relative;
+  display: inline-block;
+}
+
+/* Tooltip text */
+.tooltip .tooltiptext {
+  visibility: hidden;
+  width: 80px;
+  background-color: #000;
+  color: #fff;
+  text-align: center;
+  padding: 5px 0;
+  border-radius: 6px;
+  font-size: 12px;
+
+  /* Position the tooltip text */
+  position: absolute;
+  z-index: 1;
+  bottom: 125%;
+  left: 50%;
+  margin-left: -43px;
+
+  /* Fade in tooltip */
+  opacity: 0;
+  transition: opacity 0.3s;
+}
+
+/* Tooltip arrow */
+.tooltip .tooltiptext::after {
+  content: "";
+  position: absolute;
+  top: 100%;
+  left: 50%;
+  margin-left: -5px;
+  border-width: 5px;
+  border-style: solid;
+  border-color: #555 transparent transparent transparent;
+}
+
+/* Show the tooltip text when you mouse over the tooltip container */
+.tooltip:hover .tooltiptext {
+  visibility: visible;
+  opacity: 1;
+}
```

Comment:
I would add `justify-content: space-between;` to `.stars` class in order to justify elements, like this: 
![image](https://user-images.githubusercontent.com/47003650/140174717-d2dc28cc-2b05-41e6-bcdb-120e9bf99949.png)
Btw. shouldn't `.stars` class be updated to sth like `.contributors`? ;)

Reply:
Unfortunately it increases the problem with the last tooltip (most to the right) hidden by the blue background (screens on my previous comment). I propose to apply this change only if we fix this problem.

Reply:
I renamed .stars into .contributors :)

Reply:
With the fix above, in order to locate avatars on the right side, you could just set `margin-right` to `auto` in `.contributors__label` class

Reply:
Done. Now it looks perfect